attributes/default.rb

```diff
@@ -0,0 +1,61 @@
+#
+# Cookbook Name:: centreon-engine
+# Recipe:: default
+#
+# Copyright (c) 2016 Criteo, All Rights Reserved.
+
+#  --[ Centreon engine sudo configuration ]--
+default['centreon_engine']['sudo']['users'] = "centreon-engine"
```

Comment:
maybe you could reuse the user name attribute instead of defining a new attribute just for sudo.